lib/fs/__tests__/realpath.test.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const fs = require('fs')
+const fse = require('../..')
+const assert = require('assert')
+
+/* eslint-env mocha */
+
+// fs.realpath.native only available in Node v9.2+
+if (typeof fs.realpath.native === 'function') {
+  describe('realpath.native', () => {
+    it('works with callbacks', () => {
+      fse.realpath.native(__dirname, (err, path) => {
+        assert.ifError(err)
+        assert.equal(path, __dirname)
```

Comment:
It is better to use `assert.strictEqual()` since `assert.equal()` is deprecated in the legacy mode.